language-support/java/codegen/src/it/java-1.1/com/digitalasset/lf_1_1/ListTest.java

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset;
+
+import com.daml.ledger.javaapi.data.*;
+import com.digitalasset.ledger.api.v1.ValueOuterClass;
+import com.google.protobuf.Empty;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import tests.listtest.*;
+import tests.listtest.color.Green;
+import tests.listtest.color.Red;
+import tests.listtest.listitem.Node;
+import tests.varianttest.variantitem.ParameterizedRecordVariant;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+@RunWith(JUnitPlatform.class)
+public class ListTest {
```

Comment:
Instead of copying the file, please add this test to `java-1.0/.../AllTests.java` and `java-1.1/.../AllTests.java` next to the list of suite classes next to `AllGenericTests.class`

Reply:
Cannot do it, otherwise `lf_latest` test fails trying to compile it.